addons/transifex/models/transifex_code_translation.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from configparser import ConfigParser
+from os.path import join as opj
+import os
+import werkzeug.urls
+import psycopg2
+
+import odoo
+from odoo import models, fields, _
+from odoo.tools.translate import code_translations
+
+
+class TransifexCodeTranslation(models.Model):
+    _name = "transifex.code.translation"
+    _description = "Code Translation"
+    _log_access = False
+
+    source = fields.Text(string='Code')
+    value = fields.Text(string='Translation Value')
+    module = fields.Char(help="Module this term belongs to")
+    # lang = fields.Selection(selection='_get_languages', string='Language', validate=False)
+    lang = fields.Char(string='Language')
+    type = fields.Char(string='Type')
+    transifex_url = fields.Char("Transifex URL", compute='_get_transifex_url', store=True, precompute=True,
+                                help="Propose a modification in the official version of Odoo")
+
+    def _auto_init(self):
+        super()._auto_init()
+        self.env.cr.execute(f'DELETE FROM {self._table}')
+
+    def _load_code_translations(self, module_names=None, langs=None):
+        if module_names is None:
+            module_names = self.env['ir.module.module'].search([('state', '=', 'installed')]).mapped('name')
+        if langs is None:
+            langs = [lang for lang, _ in self.env['res.lang'].get_installed()]
+        create_value_list = []
+        self.env.cr.execute(f'SELECT DISTINCT module, lang FROM {self._table}')
+        loaded_code_translations = set(self.env.cr.fetchall())
+        for module_name in module_names:
+            for lang in langs:
+                if (module_name, lang) not in loaded_code_translations:
+                    create_value_list.extend([{
+                        'source': src,
+                        'value': value,
+                        'module': module_name,
+                        'lang': lang,
+                        'type': 'python',
+                    } for src, value in code_translations.get_python_translations(module_name, lang).items()])
```

Comment:
This only returns existing translations? We should also see empty translations ideally (where a value is missing)

Reply:
Yes, that's why I mark this one draft.
I only make it works for UI. I promise empty translations will be there:)